app/assets/javascripts/users.js.erb

```diff
@@ -118,7 +118,9 @@ function createTarget(pos) {
 function backToHome() {
   $('.home-left-new-target').addClass('hidden');
   $('.home-left-delete-target').addClass('hidden');
+  $('.edit-user-container').remove();
   $('.home-left').removeClass('hidden');
+  $('.home-left-top').removeClass('hidden');
```

Comment:
You can group selectors, for example: `$('.home-left-top, .home-left').removeClass('hidden');`

Replace in other cases if you can.